lib/r10k/settings.rb

```diff
@@ -160,7 +160,10 @@ def self.deploy_settings
               raise ArgumentError, "The specified puppet.conf #{value} is not readable"
             end
           end
-        }),
+                       }),
+        Definition.new(:deploy_spec, {
+                       :desc => "Whether or not to deploy the /spec dir of a module. Defaults to false.",
```

Comment:
The `/` was an attempt convey the root of the module, but, it seems possibly confusing, I can remove that.

I think it actually does match what's above? The above `flag` means that if it is passed like `r10k deploy environment --deploy-spec` than it will deploy the `spec` dir. In this `Definition`, it is just supposed to be default value of false.

Reply:
Oh, sorry, I just meant the way you're doing `spec`. In one place the slash was before, in the other, it was after. I think we can just drop the slash entirely.